services/ui/package.json

```diff
@@ -39,10 +39,11 @@
     "react-apollo": "^2.1.11",
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.8.4",
+    "react-hexgrid": "^1.0.3",
     "react-highlight-words": "^0.14.0",
     "react-modal": "^3.8.1",
     "react-nice-dates": "^1.0.2",
-    "react-select": "^2.1.1",
+    "react-select": "^3.0.0",
```

Comment:
Does this change the effective installed version at all?
I don't think we have to worry - I just want to know if there are ramifications to pushing up the major version?

Reply:
![image](https://user-images.githubusercontent.com/2339381/84885883-8ebe4b80-b08b-11ea-9b7c-1f38ba867748.png)
the existing selects work nicely still - this allows us to use some of the latest async stuff as well now